benchmark/utils/general.hpp

```diff
@@ -58,6 +58,10 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #include <rapidjson/prettywriter.h>
 
 
+
```

Comment:
Sorry, maybe I count the number of lines wrongly.
There are two new line between two header section.